src/main/haskell/kore/src/Kore/Exec.hs

```diff
@@ -86,7 +86,7 @@ import           Kore.Unparser
 import           Kore.Variables.Fresh
                  ( FreshVariable )
 
--- | Concrete execution
+-- | ConcreteVariable execution
```

Comment:
I think your find-and-replace went haywire here, and below.

Reply:
Yeah, I renamed more than I intended... strangely enough it still compiled